source/Octopus.Tentacle.Tests.Integration/ClientFileTransferRetriesTimeout.cs

```diff
@@ -142,22 +142,21 @@ public async Task WhenRpcRetriesTimeOut_DuringDownloadFile_TheRpcCallIsCancelled
                 .WithRetryDuration(TimeSpan.FromSeconds(15))
                 .WithPortForwarderDataLogging()
                 .WithResponseMessageTcpKiller(out var responseMessageTcpKiller)
+                .WithTcpConnectionUtilities(Logger, out var tcpConnectionUtilities)
                 .WithTentacleServiceDecorator(new TentacleServiceDecoratorBuilder()
-                    .LogCallsToFileTransferService()
-                    .CountCallsToFileTransferService(out var fileTransferServiceCallCounts)
-                    .RecordExceptionThrownInFileTransferService(out var fileTransferServiceException)
-                    .DecorateFileTransferServiceWith(d => d
-                        .BeforeDownloadFile(async (service, _) =>
+                    .RecordMethodUsages<IAsyncClientFileTransferService>(out var tracingStats)
```

Comment:
Nit: With this method, you make the out var `tracingStats`, `recordedUsages` and `scriptRecordedUsages` in different files, it might be good to make these all the same to increase clarity when skipping between tests etc.

Reply:
ah yep, good call. it was called tracing stats so let me do a cleanup

Reply:
fixed in 8d43db8d